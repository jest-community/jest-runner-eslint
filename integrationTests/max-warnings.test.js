const runJest = require('./runJest');

it('Fails if more than max warnings', () => {
  return expect(runJest('max-warnings')).resolves.toMatchSnapshot();
});
