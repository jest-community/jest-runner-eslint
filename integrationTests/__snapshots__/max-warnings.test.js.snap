// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fails if more than max warnings 1`] = `
"FAIL integrationTests/__fixtures__/max-warnings/__eslint__/file.js
  ✕ ESLint
/mocked-path-to-jest-runner-mocha/integrationTests/__fixtures__/max-warnings/__eslint__/file.js
  3:1  warning  Unexpected console statement  no-console
  4:1  warning  Unexpected console statement  no-console
  5:1  warning  Unexpected console statement  no-console
✖ 3 problems (0 errors, 3 warnings)
ESLint found too many warnings (maximum: 2).
Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:
Ran all test suites.

"
`;
